src/routes/popup/settings/quickSettings.tsx

```diff
@@ -52,111 +49,10 @@ export default function QuickSettings({ params }: Props) {
   );
 }
 
-function SettingListItem({
-  displayName,
-  description,
-  icon,
-  active,
-  ...props
-}: SettingItemData & HTMLProps<HTMLDivElement>) {
-  return (
-    <ListItem
-      title={
-        (
-          <Title>
-            {browser.i18n.getMessage(displayName)}{" "}
-            {props.isExternalLink && <ExternalLinkIcon />}
-          </Title>
-        ) as ReactNode & string
-      }
-      description={browser.i18n.getMessage(description)}
-      active={active}
-      small={true}
-      {...props}
-    >
-      <ListItemIcon as={icon} />
-    </ListItem>
-  );
-}
-
-interface Props {
-  params: {
-    setting?: string;
-    subsetting?: string;
-  };
-}
-
-interface Setting extends SettingItemData {
-  name: string;
-  component?: (...args: any[]) => JSX.Element;
-  externalLink?: string;
-}
-
-type SettingItemData = {
-  icon: Icon;
-  displayName: string;
-  description: string;
-  active: boolean;
-  isExternalLink?: boolean;
-};
-
-const ExternalLinkIcon = styled(LinkExternal02)`
-  height: 1rem;
-  width: 1rem;
-  color: ${(props) => props.theme.secondaryTextv2};
-`;
-
-const Title = styled.div`
-  display: flex;
-  align-items: center;
-  justify-items: center;
-  gap: 8px;
-`;
-
 const SettingsList = styled.div`
   position: relative;
   display: flex;
   flex-direction: column;
   gap: 1.125rem;
   padding: 0 1rem;
 `;
-
-const allSettings: Omit<Setting, "active">[] = [
```

Comment:
These where mostly duplicated (except for the `component` property) in `ArConnect/src/routes/dashboard/dashboard.constants.ts`, but now they are derived from another value in that same file.